frontend/src/components/grantaccess/UserIdentityCard.vue

```diff
@@ -2,11 +2,19 @@
 import Card from 'primevue/card';
 import Icon from '@/components/common/Icon.vue';
 import { IconSize } from '@/enum/IconEnum';
-import type { IdimProxyIdirInfo } from 'fam-app-acsctl-api';
+import type { IdimProxyBceidInfo, IdimProxyIdirInfo } from 'fam-app-acsctl-api';
+import type { PropType } from 'vue';
 
-const props = defineProps<{
-    userIdentity: IdimProxyIdirInfo;
-}>(); // Vue3 alternative way for Type the defineProps().
+const props = defineProps({
+    userIdentity: {
+        type: Object as unknown as PropType<IdimProxyBceidInfo>,
```

Comment:
maybe just `Object as PropType<IdimProxyBceidInfo>`? do we need unknown?

Reply:
Typescript was complaining and people on stack overflow suggested that but I tried without now and was ok so I'm changing. Thanks!